lib/update.js

```diff
@@ -85,6 +85,13 @@ function update(updates, object, ...args) {
 
   const resolvedUpdates = resolveUpdates(updates, defaultedObject)
 
+  // Remove the u.omitted updates on keys that aren't there,
+  // in order to avoid creating a copy of the original object
+  Object.entries(resolvedUpdates)
```

Comment:
I think you had a version using a for loop before -- I preferred that version. This one enumerates 3 times and makes the code a little harder to follow.

Reply:
Done